src/radical/pilot/pmgr/launching/default.py

```diff
@@ -536,7 +538,8 @@ def _start_pilot_bulk(self, resource, schema, pilots):
             tgt_dir = os.path.dirname(tgt)
 
             if tgt_dir.startswith('..'):
-              # raise ValueError('staging tgt %s outside pilot sbox: %s' % (ft['tgt'], tgt))
+              # raise ValueError('staging tgt %s outside pilot sbox: %s'
+              #                  % (ft['tgt'], tgt))
```

Comment:
We probably can remove this altogether by now...

Reply:
you've updated this part, should it still be removed?

Reply:
yes, I think so.

Reply:
done